### PR TITLE
fix: input amount issue

### PR DESCRIPTION
**Pull request by matt-user**
closes FRO-134

### CODE REVIEW

**Review by LuizAsFight — CHANGES_REQUESTED** [review]
this issue should be fixed in `fuel-ui` side

this [task](https://linear.app/fuel-network/issue/FRO-134/inputamount-consider-zero-and-undefined-the-same-thing) is in `Design System` Project

File: packages/app/src/systems/Ecosystem/components/ProjectItem/ProjectItem.tsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ export const ProjectItem: ProjectItemComponent = ({
           </Box.Stack>
           <Box.Flex align="center" justify="space-between" wrap="wrap">
             <ButtonLink
-              variant="link"
+              as="a"
```
Comment (LuizAsFight) [inline packages/app/src/systems/Ecosystem/components/ProjectItem/ProjectItem.tsx:86]: why change this ? looks not related to this pr objective

**Review by LuizAsFight — APPROVED** [review]

File: packages/app/src/systems/Ecosystem/components/ProjectItem/ProjectItem.tsx (hunk 1 above)
Comment (matt-user) [reply packages/app/src/systems/Ecosystem/components/ProjectItem/ProjectItem.tsx:86]: When bumping fuel-ui the `ButtonLink` component no longer accepted the variant prop.  Not sure if this is a bug or not.
